bazel/tclap-win64-ull-sizet.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/include/tclap/StandardTraits.h b/include/tclap/StandardTraits.h
```

Comment:
Can you fix this one upstream?

Reply:
We will submit it, but it is needed locally short term.

Reply:
It seems small enough that we could kick this into the pipeline in parallel. We really dislike patches, they are only a solution when all else fails when working with an upstream project.

Reply:
If and when we ever see a tclap 1.4 release, this patch and the defect go away, since the entire issue was refactored away 6 years ago;
https://sourceforge.net/p/tclap/code/ci/5d4ffbf2db794af799b8c5727fb6c65c079195ac/
Please consider this patch acceptable short-term (or permanently given the stable/no-releases state) for this component and we'll treat this as resolved for modern windows 64 compilers.

Reply:
Ack, please leave a reference to this thread in the `patches` arg of the `_repository_impl` .

Reply:
Added short note + refactor link